cmd/minikube/cmd/root.go

```diff
@@ -98,6 +99,17 @@ func Execute() {
 		} else {
 			os.Args = append([]string{RootCmd.Use, callingCmd, "--"}, os.Args[1:]...)
 		}
+	} else if runtime.GOOS == "windows" && driver.IsMicrosoftWSL() {
```

Comment:
@hetong07 why does this have to be in an else clause? could it be just a root level if statement ?

Reply:
any chance I could see example of this output in real WSL ?

Reply:
@medyagh Yes you are right. I just prefer to put all special case handler in one if-else tree. Since they are irrelevant, we should put it in a separate if-else.

The running demo is:
![image](https://user-images.githubusercontent.com/32189150/107166395-b63f8c80-696a-11eb-8d99-298fa4e9257d.png)